docker-compose.yaml

```diff
@@ -13,17 +13,30 @@ services:
       - "127.0.0.1:5000:5000"
     depends_on:
       - mongodb
+      - minio
     environment:
       - KBCOLL_KBASE_AUTH2_URL=https://ci.kbase.us/services/auth
       - KBCOLL_KBASE_AUTH2_ADMIN_ROLES=CDM_TASK_SERVICE_ADMIN
       - KBCOLL_KBASE_STAFF_ROLE=KBASE_STAFF
       - KBCOLL_HAS_NERSC_ACCOUNT_ROLE=HAS_NERSC_ACCOUNT
       - KBCOLL_SFAPI_CRED_PATH=/creds/sfapi_creds
       - KBCOLL_SFAPI_USER=cdm_ts
+      - KBCOLL_S3_URL=http://minio:9000
+      - KBCOLL_S3_ACCESS_KEY=miniouser
+      - KBCOLL_S3_ACCESS_SECRET=miniopassword
     volumes:
       - ./tmp_creds:/creds
 
-# TODO MINIO add minio
+  minio:
+    container_name: minio
```

Comment:
I think the problem here is having a container name specified, since by default docker compose prefixes the container name with the build context:

```
 ✔ Container cdm-task-service-mongodb-1           Started
1.5s
 ✔ Container cdm-task-service-minio-1             Started
2.4s
 ✔ Container cdm-task-service-cdm_task_service-1  Started
```

I've removed the container name; we should probably do the same for other projects.

Reply:
👍 removing container name makes sense.